### DISCUSSION TITLE
How to save the table?

### DISCUSSION THREAD

**Discussion by twj8CDC** [posit-dev/great-tables#206] [Q&A] [answered]
Awesome looking package!
Is there a way to save the table? Ideally, my teams workflow would be to save to excel, but even as an HTML table or convert back to pandas would work. Is this feature implemented?

**Comment by rich-iannone** [comment, marked as answer]
We now have the feature to export a table as HTML. Here is the reference page for that: https://posit-dev.github.io/great-tables/reference/GT.as_raw_html.html.
